Source/Rhetos.Configuration.Autofac/ApplicationDeployment.cs

```diff
@@ -91,14 +84,23 @@ public void GenerateApplication()
             }
         }
 
+        internal RhetosContainerBuilder RegisterComponentsForBuild()

```

Comment:
Consider 'Create' as prefix for this and similar functions for clarity (e.g. CreateBuildComponentsContainer).